registry/tests/appRoutes.spec.ts

```diff
@@ -469,6 +556,32 @@ describe(`Tests ${example.url}`, () => {
             }
         });
 
+        it('should successfully update record with existed domainId', async () => {
+            const domainId1 = await createRouterDomain(example.routerDomain);
```

Comment:
Same as above, you need to do proper cleanup in case of error

Reply:
done